framework/include/executioners/SecantSolve.h

```diff
@@ -79,27 +79,30 @@ class SecantSolve : public FixedPointSolve
   /// Print the convergence history of the coupling, at every coupling iteration
   virtual void printFixedPointConvergenceHistory() override final;
 
+  /// Default TagID if storage is not allocated
+  static TagID _default_tag_id;
```

Comment:
Few things:

- For static variables, don't prefix with an underscore. Underscore implies object state, even though this doesn't change
- These should typically be const
- While you can initialize these in the header, we typically prefer to do this in the initializer list

Fortunately - `INVALID_TAG_ID` already exists in `MooseTypes.h` and this should be used instead. So, remove this definition and declaration, use `INVALID_TAG_ID`, and set it in the initializer list instead.